addons/project/static/src/views/project_task_kanban/project_task_kanban_renderer.js

```diff
@@ -37,28 +32,10 @@ export class ProjectTaskKanbanRenderer extends KanbanRenderer {
         return canResequenceGroups;
     }
 
-    async onWillStart() {
-        if (!this.props.list.isGroupedByPersonalStages) { // no need to check it if the group by is personal stages
-            this.isProjectManager = await this.userService.hasGroup('project.group_project_manager');
-        }
-    }
-
-    isProjectTasksContext() {
-        return this.props.list.context.active_model === "project.project" && this.props.list.context.default_project_id;
-    }
-
-    canCreateGroup() {
-        return (super.canCreateGroup() && this.isProjectTasksContext() && this.props.list.isGroupedByStage) || this.props.list.isGroupedByPersonalStages;
-    }
-
     canDeleteGroup(group) {
         return super.canDeleteGroup(group) && (!this.props.list.isGroupedByStage || this.isProjectManager) || this.props.list.isGroupedByPersonalStages;
     }
 
-    canEditGroup(group) {
-        return super.canEditGroup(group) && (!this.props.list.isGroupedByStage || this.isProjectManager) || this.props.list.isGroupedByPersonalStages;
-    }
-
     async deleteGroup(group) {
```

Comment:
I think we should also move that method otherwise we will have 2 wizards when the user want to remove a column when the group by is `stage_id`. 😬

Reply:
wym, it has been moved :thinking: